subsys/shell/Kconfig

```diff
@@ -237,6 +237,12 @@ config SHELL_LOG_BACKEND
 	  using the shell backend's LOG_LEVEL option
 	  (e.g. CONFIG_SHELL_TELNET_INIT_LOG_LEVEL_NONE=y).
 
+config SHELL_STRING_CONV
```

Comment:
Not sure if a separate Kconfig option is required in this case. The functions will anyway be removed from the code if they are not used.